src/lib/marineCli.ts

```diff
@@ -62,16 +63,28 @@ export const initMarineCli = async (
     cwd,
     printOutput = true,
   }): Promise<string> => {
-    return execPromise({
-      command: marineCLIPath,
-      args,
-      flags,
-      spinnerMessage:
-        message === undefined
-          ? undefined
-          : getMessageWithKeyValuePairs(message, keyValuePairs),
-      options: { cwd },
-      printOutput,
-    });
+    try {
+      return await execPromise({
+        command: marineCLIPath,
+        args,
+        flags,
+        spinnerMessage:
+          message === undefined
+            ? undefined
+            : getMessageWithKeyValuePairs(message, keyValuePairs),
+        options: { cwd },
+        printOutput,
+      });
+    } catch (e) {
+      if (
+        isObject(e) &&
+        hasKey("message", e) &&
+        typeof e["message"] === "string"
+      ) {
```

Comment:
Maybe have a custom type for errors and do `instance of MyError`?

Reply:
It doesn't matter what is the type of the error here I think. if error has a message - then show it. The change is about treating marine errors as exepected errors so we don't show stack traces when marine command fails

Reply:
I imporoved that part, but this "message" will never change. This is what happens when you do new Error() in js - it always creates an object with a "message" property in it. CLIError extends default js error but is the same

Reply:
if you do `instanceof CLIError`, then you can access message as a field, via `.`, right?

Reply:
It's true, but it doesn't matter what kind of error we have. Any error with a message is good. I added a genral check for that without using isntanceof, please check it out